source/h2o_modelling.py

```diff
@@ -41,20 +41,22 @@ def prepare_h2o_df(df, outcome, to_factor=True):
                             Defaults to True. 
 
     Returns:
-        tuple: A tuple with two elements. The first is the name of the outcome variable and the second is a list of 
-        predictor variables.
+        tuple: A tuple with three elements: the h2o data frame, the name of the outcome variable, 
+        and the second is a list of predictor variables.
     """
     if to_factor:
-        df[outcome].asfactor()
+        df[outcome] = df[outcome].asfactor()
 
     y = outcome
-    x = df.columns.remove(y)
+    x = df.columns
+    x.remove('label')
+    x.remove('name')
 
-    return (y, x)
+    return (df, y, x)
 
 
-def train_and_save(df, outcome, predictors, save_path, max_models=100, max_runtime_min=5):
-    """Trains and saves an h2o model using H2OAutoML.
+def train_and_save(df, outcome, predictors, save_path, max_models=100, max_runtime_min=5, seed=1):
+    """Trains and saves an h2o model using H2OAutoML. Excludes deep learning models (see README for more detail.)
```

Comment:
@ZacharyHimmelberger Was the README updated in a separate PR?

Reply:
I just added it. Further, this description is in the wrong function. I also fixed that.